goalgorilla/open_social
# Feature/comments readability

This PR improves the margin targetting of comment paragraphs 
and adds some updated icons:

It looks better if the like button is flush against the bottom of
a comment. To achieve this the margin-bottom of paragraphs in
comment text is set to 0. This has the drawback that long
paragraphs become very difficult to read.

This commit changes the targetting of the margin-bottom to be only
for the last paragraph in a comment. This ensures that for both
single paragraph and multi-paragraph comments the like button is
flush against the last paragraph. However it ensures that spacing
between multiple paragraphs is retained.

This changes nothing for short, one paragraph comments.

## How to test
- Confirm that after the change short paragraphs don't look any different
- Confirm that longer comments with multiple paragraphs are now spaced and readable.
